forge16/build.gradle

```diff
@@ -44,17 +48,17 @@ repositories {
 
 dependencies {
     minecraft 'net.minecraftforge:forge:1.16.5-36.2.34'
-    implementation 'pixelmon:Pixelmon-1.16.5-9.0.2-server:9.0.2'
+    implementation 'pixelmon:Pixelmon-1.16.5-9.1.2-server:9.1.2'
 
     compileOnly 'com.github.EnvyWare.ForgePlaceholderAPI:api:2.0.4'
 
     shadow group: 'org.spongepowered', name: 'configurate-yaml', version: '4.0.0'
 
-    shadow (group: 'com.github.EnvyWare.API', name: 'commons', version: '2.7.5')
-    shadow (group: 'com.github.EnvyWare.API', name: 'forge16', version: '2.7.5') {
+    shadow (group: 'com.envyful.api', name: 'commons', version: '4.2.6')
+    shadow (group: 'com.envyful.api', name: 'forge16', version: '4.2.6') {
         transitive = false
     }
-    shadow (group: 'com.github.EnvyWare.API', name: 'reforged16', version: '2.7.5') {
+    shadow (group: 'com.envyful.api', name: 'reforged16', version: '4.2.6') {
```

Comment:
If you're going to update  the API please make sure it's on the latest version

Reply:
I will update the API to the latest version as soon as possible and push the commit.